lib/selector/parser/ast.go

```diff
@@ -63,6 +105,10 @@ func (sel selectorRoot) EvaluateLabels(labels Labels) bool {
 	return sel.root.Evaluate(labels)
 }
 
+func (sel *selectorRoot) AcceptVisitor(v Visitor) {
+	sel.root.AcceptVisitor(v)
+}
+
 func (sel selectorRoot) String() string {
```

Comment:
If we're starting to use these structs as pointers, we should change all the methods to have pointer receivers.

Reply:
ok, wasn't sure if that was desired or not - the other methods don't modify the node, so thought keeping the same was OK.